cgmml/common/evaluation/CV/evaluation.py

```diff
@@ -156,6 +164,54 @@ def process_height_prediction(depthmap_file: str, rgb_file: str, calibration_fil
     return height, angle
 
 
+def run_evaluation(metadata: dict, calibration_file: str):
+
+    avg_err = 0
+    last_fail = 0
+    output = []
+    rejections = []
+    keys = metadata.keys()
+    size = len(keys)
+    for key_index, key in enumerate(keys):
+        logger.info('Processing %d/%d', key_index + 1, size)
```

Comment:
Optional suggestion: Could be simplified to:
```suggestion
    for key in tqdm(keys):
```

Reply:
tqdm on my system really breaks the logging,it is hard to read it. I will avoid using it